src/project-tests/quote-machine-tests.js

```diff
@@ -123,12 +139,21 @@ export default function createRandomQuoteMachineTests() {
       it(`When the #new-quote button is clicked, my quote machine
       should fetch a new quote and display it in the #text element.`, function () {
         let prevText;
+
         const newQuoteBtn = document.getElementById('new-quote');
+        newQuoteBtn.click();
```

Comment:
I think this needs to be moved below the `prevText = ...` (line 151) - or else `prevText` and `newText` would always be the same no?